features/dbPing.js

```diff
@@ -0,0 +1,58 @@
+var mysql = require("mysql2");
+
+module.exports = function (app) {
+  app.message("dbping", dbPing);
+};
+
+async function dbPing({ message, client }) {
+  // Connect to the database
```

Comment:
Lines 8-29  seem fine functionally, but I think they'll make a new DB connection every time this functions run. If you want, you could try connecting to the DB outside the function, or in a different file, so you only connect once, and then reuse the established connection.

Not required, just an option if you'd like to try it out.

Reply:
We'll definitely want to maintain a connection pool and grab a connection from the pool whenever we need to make a query

https://www.npmjs.com/package/mysql2#using-connection-pools